### PR TITLE
[doc][ybm] New Free trial

### PR DESCRIPTION
**Pull request by ddhodge**
New free trial flow
DOC-210

@netlify /preview/yugabyte-cloud/managed-freetrial/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-freetrial.md [hunk 1]
```diff
@@ -11,26 +11,21 @@ menu:
 type: docs
 ---
 
-To go beyond the capabilities of the [Sandbox cluster](../cloud-basics/create-clusters/create-clusters-free/), request a free trial. If you qualify, you will receive free credits towards creating Dedicated clusters and accessing other features in YugabyteDB Managed.
+To go beyond the capabilities of the [Sandbox cluster](../cloud-basics/create-clusters/create-clusters-free/), start a free trial to receive free credits towards creating Dedicated clusters and exploring other features in YugabyteDB Managed.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-freetrial.md:14]: Add link to dedicated clusters.

File: docs/content/preview/yugabyte-cloud/managed-freetrial.md [hunk 2]
```diff
@@ -39,6 +34,11 @@ After the free trial begins, you can use your credits to create dedicated cluste
...
 - Read replicas
 
-Create a [billing profile](../cloud-admin/cloud-billing-profile/) and add a payment method any time. You won't be charged until your credits are used.
+[Choose a plan](https://www.yugabyte.com/yugabyte-cloud-standard-price-list/) any time. You won't be charged until your credits are used.
+
+## Limitations
 
-Only one free trial can be applied to an account.
+- Only one free trial can be applied to an account.
+- The free trial features the Professional plan. Enterprise plan features are not included.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-freetrial.md:42]: what are these plans ? Can you add details about whats included in them ?

**Review by aishwarya24 — APPROVED** [review]
LGTM.

File: docs/content/preview/yugabyte-cloud/managed-freetrial.md [hunk 3]
```diff
@@ -39,6 +34,11 @@ After the free trial begins, you can use your credits to create dedicated cluste
...
+- Not all regions are available during free trials.
```
Comment (Arnav15) [inline docs/content/preview/yugabyte-cloud/managed-freetrial.md:43]: @shreyas-agnihotri : do we also want to call out that we will only allow 24 cores maximum during the free trial period?

**Pull request merged**